.github/workflows/python.yml

```diff
@@ -1,31 +0,0 @@
-name: Python Checks
```

Comment:
Is this workflow being removed in favor of using the pre-commit method?  If so, I would prefer to keep this one here in the classic GHA sense in case contributors are not relying on the pre-commit piece.

Reply:
Oh, ok.  That's my misunderstanding.  In that case, kill it.